tekton/templates/251-bot-serviceaccount.yaml

```diff
@@ -21,4 +22,5 @@ secrets:
 {{- end }}
 {{- if .Values.auth.docker.username }}
```

Comment:
@jstrachan I believe we don't use `Values.auth.docker` with jenkins-x, can you confirm?

Reply:
coz, with eksctl it doesn't add `knative-git-user` variables in the created SA

Reply:
I don't think we do - but this would allow someone to inject a custom docker secret if they wanted I think via a custom configuration - e.g. via https://jenkins-x.io/docs/labs/boot/apps/#customising-charts

typically when folks use AWS/Azure/GCP we expect them to use the cloud well and use ECR/ACR/GCR - but some folks may have additional container registries + associated secrets.

Am sure we could improve this more though ;)